recipes/cgal/all/conanfile.py

```diff
@@ -45,22 +48,26 @@ def _configure_cmake(self):
             self._cmake.definitions["WITH_CGAL_Core"] = self.options.with_cgal_core
             self._cmake.definitions["WITH_CGAL_Qt5"] = self.options.with_cgal_qt5
             self._cmake.definitions["WITH_CGAL_ImageIO"] = self.options.with_cgal_imageio
-            self._cmake.definitions["CGAL_HEADER_ONLY"] = self.options.header_only
+            if "header_only" in self.options:
+                self._cmake.definitions["CGAL_HEADER_ONLY"] = self.options.header_only
```

Comment:
```suggestion
            self._cmake.definitions["CGAL_HEADER_ONLY"] = self.options.get_safe("header_only", False)
```
Keep get_safe. In case that option does not exist, cmake will show a warning message only.

Reply:
Since CGAL-5.3, `CGAL_HEADER_ONLY` is no longer a CMake option of CGAL. That is why I wanted to avoid a CMake warning about an unused command line argument. I see that the conan build of CGAL already has other warnings about unused variables. I will do as you recommend.

Reply:
@lrineau Makes sense. I just updated the suggestion following the regular pattern with get safe.